porch/pkg/git/git_test.go

```diff
@@ -96,7 +96,7 @@ func (g GitSuite) TestOpenEmptyRepository(t *testing.T) {
 		t.Errorf("Unexpectedly succeeded opening empty repository with main branch validation enabled.")
 	}
 
-	if _, err := OpenRepository(ctx, name, namespace, repository, tempdir, GitRepositoryOptions{SkipMainBranchVerification: true}); err != nil {
+	if _, err := OpenRepository(ctx, name, namespace, repository, tempdir, GitRepositoryOptions{MainBranchStrategy: SkipVerification}); err != nil {
```

Comment:
Can we also test the other `MainBranchStrategy` options here?

Reply:
Added.